src/Repository/CommitteeRepository.php

```diff
@@ -334,4 +337,32 @@ public function paginate(int $offset = 0, int $limit = SearchParametersFilter::D
 
         return new Paginator($query);
     }
+
+    public function retrieveTopCommitteesInReferentManagedArea(Adherent $referent, $mostActive = true): array
+    {
+        if (!$referent->isReferent()) {
+            throw new \InvalidArgumentException('Adherent must be a referent.');
+        }
+
+        return $this->createQueryBuilder('committee')
+            ->select('committee.name, COUNT(event) AS events, SUM(event.participantsCount) as participants')
+            ->join(Event::class, 'event', Join::WITH, 'event.committee = committee.id')
+            ->join('committee.referentTags', 'tag')
+            ->where('tag.id IN (:tags)')
+            ->andWhere('committee.status = :status')
+            ->andWhere('event.beginAt >= :firstDayOfThisMonth')
+            ->andWhere('event.beginAt < :firstDayOfNextMonth')
+            ->setParameter('tags', $referent->getManagedArea()->getTags())
+            ->setParameter('status', Committee::APPROVED)
+            ->setParameter('firstDayOfThisMonth', (new Chronos('first day of this month'))->setTime(0, 0, 0))
```

Comment:
IMHO, it's easier to understand with this naming :

~~`firstDayOfThisMonth`~~`from`
~~`firstDayOfNextMonth`~~`until`